scripts/tests_on_alpine.sh

```diff
@@ -5,6 +5,7 @@ set -e
 docker run --entrypoint sh -v "$(pwd)":/wrk -w /wrk alpine -c "
 set -e
 ./scripts/install_zig.sh
+zig/zig fmt . --check
```

Comment:
Instead of repeating the `zig fmt` check here and `tests_on_ubuntu.sh`, `release.yml`, and `test_on_macos`, could you remove those and just add a new job to `docker.yml`?

```yaml
  test_zig_format:
    name: Check "zig fmt"
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - run: ./scripts/install_zig.sh
      - run: zig/zig fmt . --check
```

Then to make it required for merge, you can add `test_zig_format` to the `needs` list of the `test` job (currently it only requires `test_on_linux`).

This way it will show up in its own section in PRs, so it is easy to spot the different between "oops, I forgot to run 'zig fmt'", and "oops, the tests are failing".